README.md

```diff
@@ -54,6 +54,7 @@ You can find Korean video lectures and presentations [here](./contents_kr.md).
 * Documentation
   * Klaytn Docs ([EN](https://docs.klaytn.com/), [한](https://ko.docs.klaytn.com/))
   * [Tech Blog](https://medium.com/klaytn/tagged/technology)
+   * [Tech Blog Article List](https://github.com/klaytn/klaytn/wiki/Tech-Blog)
```

Comment:
```suggestion
 * [Tech Blog Article List](./tech-blog.md)
```

I think we don't need to provide the tech blog link since we have an article list here.

Reply:
... ㅋㅋ stupid me